webapp/channels/src/components/post/post_options.tsx

```diff
@@ -120,12 +120,14 @@ const PostOptions = (props: Props): JSX.Element => {
     let commentIcon;
     if (showCommentIcon) {
         commentIcon = (
-            <CommentIcon
-                handleCommentClick={props.handleCommentClick}
-                postId={post.id}
-                extraClass={commentIconExtraClass}
-                commentCount={props.collapsedThreadsEnabled ? 0 : props.replyCount}
-            />
+            <li key='commentIcon__listItem'>
```

Comment:
Are you sure that we need these keys on each `li` tag? React only requires these when you render an array of components, so I don't think that they're actually needed for most of these
```suggestion
            <li>
```